conf/messages.en

```diff
@@ -6,6 +6,10 @@ site.continue = Continue
 site.back = Back
 site.save_and_continue = Save and continue
 site.change = Change
+site.change.hint.disassociate.ucr = Change DUCR dissociation
+site.change.hint.associate.ucr = Change DUCR association
+site.change.hint.associate.mucr = Change MUCR association
```

Comment:
If we're splitting associate to two different messages (not using one and passing argument. The first one should be probably `ducr` not `ucr`.

```
site.change.hint.associate.ducr = Change DUCR association
site.change.hint.associate.mucr = Change MUCR association
```
or
`site.chage.hint.associate.ucr = Change {0} association`

Reply:
Good suggestion. Done.